tests/kubernetes/application/test_controller_wrapper.py

```diff
@@ -167,74 +167,95 @@ def test_sync_horizontal_pod_autoscaler_no_autoscaling():
     # Do nothing
     config_dict = {"instances": 1}
     app = setup_app(config_dict, False)
-    app.sync_horizontal_pod_autoscaler(kube_client=mock_client)
-    assert (
-        mock_client.autoscaling.create_namespaced_horizontal_pod_autoscaler.call_count
-        == 0
-    )
-    assert (
-        mock_client.autoscaling.replace_namespaced_horizontal_pod_autoscaler.call_count
-        == 0
-    )
-    assert app.delete_horizontal_pod_autoscaler.call_count == 0
+    with mock.patch(
+        "paasta_tools.kubernetes.application.controller_wrappers.autoscaling_is_paused",
+        autospec=True,
+    ) as mock_autoscaling_is_paused:
+        mock_autoscaling_is_paused.return_value = True
```

Comment:
Should probably be `False` (not paused), so the test will only delete the HPA if there is actually no autoscaling configured.

also, nit: Maybe patch as a decorator instead of a context manager.